src/settings.js

```diff
@@ -385,7 +385,7 @@ var FORCE_FILESYSTEM = 0; // Makes full filesystem support be included, even if
 var NODERAWFS = 0; // This dynamically disables virtual filesystem when the build result runs on Node.js
                    // and directly depends on behaviors of Node.js File System API.
                    // The initial working directory will be same as process.cwd() instead of VFS root directory.
-                   // The VFS becomes only available for non-Node.js environment.
+                   // The build result will throw when it runs on non-Node.js environment.
```

Comment:
This one? @kripken

Reply:
Yeah, but sorry, I should have been more clear. It's the first statement I think is confusing,

> "This dynamically disables virtual filesystem when the build result runs on Node.js"

How about text like "This mode is intended for use with Node.js (and will throw if the build is run in another engine)" at the start, removing "when the build runs on Node.js" (since that is the only intended use).

Reply:
Great, just pushed a commit for that 😃